samples/bluetooth/nrf_dm/README.rst

```diff
@@ -78,6 +78,15 @@ The addresses of these devices are stored in a list.
 When a device does not respond within a certain period of time, it is removed from the list.
 Also the last measurement results with the peers are stored in this list.
 
+Zero-distance calibration
+*************************
+
+The measured values of distance measurement include an offset.
+The offset value may depend on the design of the radio circuit, the antenna used, or the PCB layout.
+When using a library and a sample, you need to compensate the zero-distance offset and perform the calibration manually.
+To determine the offset value, place the devices at a reference distance of 60 cm from each other and collect a batch of measurement data.
+The difference between the averaged measurement value and the actual distance is the offset.
```

Comment:
I would say "The difference between the averaged measurement value and the actual distance can be used as an estimate of the offset."

Reply:
ACK